docs/intro/authoring.md

```diff
@@ -0,0 +1,36 @@
+# Workflow Authoring
+
+By opening Shift you will be prompted to an empty board.  
+
+## Creating Your First Node
+
+To create a new operator, you can either press the `Tab Key` or `Double Left-Click` with your mouse.  
+
+![Create Node Dialog](../images/node_creation_dialog.png "Create Node Dialog")  
+
+Then you can choose the type of operator you want to create and double-click on its name to create it.
+
+## Creating Connections
+
+If you want a certain operator to pass information to another one, you can connect their plugs. To do that, you just need to `Left-Click and Drag` from one plug to the other plug you want to connect to. This will propagate whatever data is produced by the first operator and pass it to the connected plug.  
```

Comment:
```suggestion
If you want a certain operator to pass information to another one, you can connect their [plugs](intro/terminology.md/#plug). To do that, you just need to `Left-Click and Drag` from one plug to the other plug you want to connect to. This will propagate whatever data is produced by the first operator and pass it to the connected plug.  
```

Reply:
Done!